.github/workflows/km-ci-workflow.yaml

```diff
@@ -114,7 +115,7 @@ jobs:
       - run: make -C payloads push-testenv-image
 
       - name: Create payloads runenv and demo-runenv images
-        run: make -C payloads runenv-image demo-runenv-image
+        run: make -C payloads runenv-image
 
       - run: make -C payloads push-demo-runenv-image
```

Comment:
This looks weird.
We say we are building demo-runenv-image but we aren't anymore, and we are pushing the demo-runenv-image we are no longer building.

Reply:
`make runenv-image` now (for a while actually) builds both runenv and demo-runenv images. Maybe a comment would help?